web/build.rs

```diff
@@ -16,18 +16,21 @@ fn main() {
     ] {
         println!("cargo:rerun-if-changed={tracked_file}");
     }
-    let vite_status = Command::new("yarn")
+
+    let mut vite_command = Command::new("yarn");
+    vite_command
         .arg("run")
         .arg("build")
         .arg("--outDir")
         .arg(&vite_out_dir)
         .arg("--base")
-        .arg("/ui/")
-        .status()
-        .unwrap();
-    if !vite_status.success() {
-        panic!("web-ui build failed: {vite_status}");
-    }
+        .arg("/ui/");
+    let vite_status = vite_command.status();
```

Comment:
suggestion: Add a newline here to give a little more breathing room.

```suggestion
        .arg("/ui/");

    let vite_status = vite_command.status();
```